content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -47,7 +48,8 @@ Check on the status of the Job with `kubectl`:
 ```shell
 kubectl describe jobs/pi
 ```
-```
+The output is similar to this:
+```shell
```

Comment:
This line is incorrect
It should be: <tt>\`\`\`</tt>

Reply:
Thanks for your guidance.